app/models/manageiq/providers/cloud_manager/orchestration_stack.rb

```diff
@@ -16,4 +19,13 @@ def self.create_stack(orchestration_manager, stack_name, template, options = {})
                  :cloud_tenant           => tenant,
                  :orchestration_template => template)
   end
+
+  def self.orchestration_stack_class_factory(orchestration_manager, template)
```

Comment:
I could see having a provider for NFVO especially since one of the central ideas is to support cross cloud.  I don't think that Tacker itself would be a provider, though.